doc/cascadia/profiles.schema.json

```diff
@@ -457,109 +458,31 @@
-              },
-              "cyan": {
-                "$ref": "#/definitions/Color",
-                "description": "Sets the color used as ANSI cyan."
-              },
-              "foreground": {
-                "$ref": "#/definitions/Color",
-                "description": "Sets the foreground color of the color table."
-              },
-              "green": {
-                "$ref": "#/definitions/Color",
-                "description": "Sets the color used as ANSI green."
-              },
-              "purple": {
-                "$ref": "#/definitions/Color",
-                "description": "Sets the color used as ANSI purple."
-              },
-              "red": {
-                "$ref": "#/definitions/Color",
-                "description": "Sets the color used as ANSI red."
-              },
-              "white": {
-                "$ref": "#/definitions/Color",
-                "description": "Sets the color used as ANSI white."
-              },
-              "yellow": {
-                "$ref": "#/definitions/Color",
-                "description": "Sets the color used as ANSI yellow."
-              }
-            },
-            "type": "object"
-          },
-          "type": "array"
-        },
         "commandline": {
           "description": "Executable used in the profile.",
           "type": "string"
         },
         "connectionType": {
           "$ref": "#/definitions/ProfileGuid",
-          "description": "A GUID reference to a connection type. Currently undocumented as of 0.3, this is used for Azure Cloud Shell"
+          "description": "A GUID reference to a connection type."
         },
         "cursorColor": {
-          "$ref": "#/definitions/Color",
-          "description": "Sets the cursor color of the profile. Overrides cursor color set in color scheme if colorscheme is set. Uses hex color format: \"#rrggbb\"."
+          "oneOf": [
+            { "$ref": "#/definitions/Color" },
+            {"type": "null"}
+          ],
+          "description": "Sets the cursor color of the profile. Overrides the cursor color set in the color scheme. Uses hex color format: \"#rrggbb\"."
         },
         "cursorHeight": {
           "description": "Sets the percentage height of the cursor starting from the bottom. Only works when cursorShape is set to \"vintage\". Accepts values from 25-100.",
           "maximum": 100,
           "minimum": 25,
-          "type": "integer"
+          "type": "integer",
+          "default": 25
         },
         "cursorShape": {
           "default": "bar",
-          "description": "Sets the cursor shape for the profile. Possible values: \"vintage\" ( ▃ ), \"bar\" ( ┃, default ), \"underscore\" ( ▁ ), \"filledBox\" ( █ ), \"emptyBox\" ( ▯ )",
+          "description": "Sets the cursor shape for the profile. Possible values:\n -\"bar\" ( ┃, default )\n -\"emptyBox\" ( ▯ )\n -\"filledBox\" ( █ )\n -\"underscore\" ( ▁ )\n -\"vintage\" ( ▃ )",
```

Comment:
For the record, I'm proud of this one:
![image](https://user-images.githubusercontent.com/11050425/79016706-81815f80-7b24-11ea-9d47-12c3fb9645ba.png)